autometrics-macros/src/lib.rs

```diff
@@ -377,3 +389,121 @@ histogram_quantile(0.95, {latency})"
 fn concurrent_calls_query(gauge_name: &str, label_key: &str, label_value: &str) -> String {
     format!("sum by (function, module) {gauge_name}{{{label_key}=\"{label_value}\"}}")
 }
+
+fn derive_get_label_impl(input: DeriveInput) -> Result<TokenStream> {
+    let variants = match input.data {
+        Data::Enum(DataEnum { variants, .. }) => variants,
+        _ => {
+            return Err(syn::Error::new_spanned(input, "#[derive(LabelValues}] is only supported for enums"));
+        },
+    };
+
+    let label_key = {
+        let attrs: Vec<_> = input.attrs.iter().filter(|attr| attr.path.is_ident("autometrics")).collect();
+
+        let key_from_attr = match attrs.len() {
+            0 => None,
+            1 => get_label_attr(attrs[0], "label_key")?,
+            _ => {
+                let mut error =
+                    syn::Error::new_spanned(attrs[1], "redundant `autometrics(label_value)` attribute");
```

Comment:
nit: import the error at the top of the file
```suggestion
                    Error::new_spanned(attrs[1], "redundant `autometrics(label_value)` attribute");
```